src/media-segment-request.js

```diff
@@ -223,16 +233,15 @@ const handleSegmentResponse = ({
 
   // This is likely an FMP4 and has the init segment.
   // Run through the CaptionParser in case there are captions.
-  if (captionParser && segment.map && segment.map.bytes) {
+  if (captionParser && segment.map && segment.map.track && segment.map.track.video) {
```

Comment:
does this ensure we have init segment data already?

Reply:
yeah